bioptim/limits/penalty.py

```diff
@@ -930,7 +930,7 @@ def custom(penalty: PenaltyOption, nodes: Union[PenaltyNodes, list], **parameter
                     nlp = nodes[0].nlp if penalty.node != Node.TRANSITION else None
                     penalty.type.get_type().add_to_penalty(node[0].ocp, nlp, val, penalty)
```

Comment:
Please make sure this line also works

Reply:
It works, but where do we have to display this kind of information in the graph ? To which nlp is it attached ?

Reply:
Done